frontend/src/layouts/DefaultLayout.test.ts

```diff
@@ -189,35 +189,67 @@ describe('DefaultLayout', () => {
   //         toStorageString: () => 'toStorageString',
   //       })
   //     })
-  //
+
   //     describe('without error', () => {
   //       beforeEach(async () => {
   //         await wrapper.find('button.user-info').trigger('click')
   //         await flushPromises()
   //         await wrapper.findComponent(UserDropdown).find('button.sign-out').trigger('click')
   //       })
-  //
+
   //       it('calls auth service sign out', () => {
   //         expect(authServiceSpy).toHaveBeenCalledWith()
   //       })
-  //
+
   //       it('clears the store', () => {
   //         expect(storeSpy).toHaveBeenCalledWith()
   //       })
   //     })
-  //
+
   //     describe('with error', () => {
   //       beforeEach(async () => {
   //         authServiceSpy.mockRejectedValue('Error!')
   //         await wrapper.find('button.user-info').trigger('click')
   //         await flushPromises()
   //         await wrapper.findComponent(UserDropdown).find('button.sign-out').trigger('click')
   //       })
-  //
+
   //       it('logs the error', () => {
   //         expect(errorHandlerSpy).toHaveBeenCalledWith('auth error', 'Error!')
   //       })
   //     })
   //   })
   // })
+
+  describe('DreamMallButton Desktop', () => {
+    it('shows the Buttonlist and hides the drawer', async () => {
+      await wrapper.find('.test-desktop-camera-button').trigger('click')
+      await wrapper.find('#dream-mall-button').trigger('click')
+      expect(wrapper.find('.button-list').classes()).toContain('button-list--active')
+      expect(wrapper.find('.v-navigation-drawer').classes()).not.toContain(
+        '.v-navigation-drawer--active',
+      )
+    })
+    it('hide the buttonlist after two clicks', async () => {
+      await wrapper.find('#dream-mall-button').trigger('click')
+      await wrapper.find('#dream-mall-button').trigger('click')
+      expect(wrapper.find('.button-list').classes()).not.toContain('button-list--active')
```

Comment:
I think I get it, you want to toggle the button menu. Feel free to add one more assertion:

```suggestion
      await wrapper.find('#dream-mall-button').trigger('click')
      expect(wrapper.find('.button-list').classes()).toContain('button-list--active')
      await wrapper.find('#dream-mall-button').trigger('click')
      expect(wrapper.find('.button-list').classes()).not.toContain('button-list--active')
```